.gitignore

```diff
@@ -0,0 +1,6 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+*.db
+*.opendb
```

Comment:
This `.gitignore` file is not needed. Please remove.

Reply:
I had to add it because of locked file.
I was pushing via Visual Studio. I will push from command line and won't need that file anymore